### ISSUE TITLE
House keeping before new release

### ISSUE THREAD

**Issue by Michel-Heinz** [Molara-Lab/Molara#387]
We plan on publishing a new release soon, but we need to clean up a bit before. I ll comment with a checklist of things that still need to be done and everyone can comment on this. I'll add the items to the list accordingly.

**Comment by Michel-Heinz** [comment]
- [ ] Check all dialog windows for titles
- [ ] Merge atom numbering and check for crystal support
- [ ] Save the settings in ~/.molara/Settings. How to uninstall? Upon installation copy the default settings file to this directory
- [ ] Restructure structure widget: This file is very large and contains functions that do not belong there.

**Comment by adrianusler** [comment]
>     * [ ]  Restructure structure widget: This file is very large and contains functions that do not belong there.

Could you be more specific? I think that the functions with regard to the measurement dialog and the builder dialog might be moved elsewhere.

**Comment by Michel-Heinz** [comment]
I would like to postpone for some days. I did not get the work done I had in mind.

**Comment by adrianusler** [comment]
@Michel-Heinz makes sense. But, we should keep in mind that it's just a release. If we don't get everything done, that's also fine, I would say.
